xml/containers-docker-dockerizing-application.xml

```diff
@@ -356,15 +354,15 @@ CMD ["-D", "FOREGROUND"]</screen>
     </callout>
     <callout arearefs="co-dockerfile-apache2-echo">
      <para>
-      Test line for debugging purposes, can be removed if everything
-      works as expected.
+      Test line for debugging purposes, can be removed if everything works as
+      expected.
      </para>
     </callout>
     <callout arearefs="co-dockerfile-apache2-copy">
      <para>
-      The copy instruction to copy own data to the
-      server's directory. The leading hash character (<literal>#</literal>)
-      marks this line as a comment, so it is not executed.
+      The copy instruction to copy own data to the server's directory. The
```

Comment:
This line is not grammatical & is unclear to me. Copy what data, from where? Whose "own"?